community_server/webroot/.htaccess

```diff
@@ -1,5 +1,6 @@
 <IfModule mod_rewrite.c>
     RewriteEngine On
+    RewriteCond %{REQUEST_FILENAME} !^/vue
```

Comment:
This .htaccess files came from cakephp default setup and are only used witch apache webserver. 
You need only to change it if you plan to support also apache webserver.

Reply:
This `.htaccess` file is read by nginx and interferes with picture delivery. It intercepts every file request (png, gif, ... most likely also suff like .zip) and routes it to the `webroot/` folder of the community server. Therefore I explicitly ignore every path starting with `/vue` to be handled by the nginx configuration itself

Reply:
Is this a gues from you?
Nginx doesn't parse .htaccess files. Trust me. 
And to prevent expose secure critical data, it is recommended to block .htaccess files in nginx config. 
What I have done with this lines of code:
```nginx 
location ~ /\.ht {
    deny  all;
}
```
Der Zugriff auf alle Dateien die mit .ht beginnen wird blockiert. 

I have remove you change in my local setup and test it. It works the same way as with this line.

Reply:
https://stackoverflow.com/questions/35766676/how-can-i-use-an-htaccess-file-in-nginx

Reply:
I see that, but I tried it:
I don't get images(vue frontend) served when I have this `.htaccess` active. I do get images served when I don't have it. Since I did not want to remove it I implemented an exception into the `.htaccess`. Maybe it was an unforseen sideffect, but my tests say otherwise.

Reply:
Okay, than we keep it that your tests keep working.

Reply:
You are correct, I could not reproduce the broken images and therefore removed the change on `.htaccess` thanks ❤️